cypress/page/index.js

```diff
@@ -0,0 +1 @@
+export { MenuContentPage, TShirtPage, SignInPage, AddresPage, ShippingPage, PaymentPage } from "./*";
```

Comment:
you can split into several lines (note that you should change TShirtPage to be ProductsListPage):
```suggestion
export { 
  MenuContentPage,
  ProductsListPage,
  SignInPage,
  AddresPage,
  ShippingPage,
  PaymentPage
} from "./*";
```